knox/views.py

```diff
@@ -79,3 +79,30 @@ def post(self, request, format=None):
         user_logged_out.send(sender=request.user.__class__,
                              request=request, user=request.user)
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+
+class TokenRefreshView(APIView):
+
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+    AUTO_REFRESH = False
+
+    def post(self, request, format=None):
+
+        if not knox_settings.ENABLE_REFRESH_ENDPOINT:
+            return Response(
+                status=status.HTTP_403_FORBIDDEN
```

Comment:
Maybe adding a message here would be nice:

```
return Response(
    data={"error": "This view is not enabled"},
    status=status.HTTP_403_FORBIDDEN,
)
```
Not sure about the data returned's format though...

Reply:
I am not sure I agree -- I dont think configuration specific messages for the library should be exposed to the users here, no value to enduser imo. Are you sure?

Reply:
I do agree that exposing config related stuff is not a good practice.
Also if I'd get a 403 from a view I'd start looking around first into my authentication / permission system not at my config...
I was trying to reduce a bit of the indirection here - not gonna block or anything - just wanted to make it easier to reason about.